README.md

```diff
@@ -0,0 +1,121 @@
+Autopilot Pattern Docker Registry
+=================================
+
+*A private Docker registry with automatic SSL support implemented according to the [Autopilot Pattern](http://autopilotpattern.io/) for automatic discovery and configuration.*
+
+[![DockerPulls](https://img.shields.io/docker/pulls/autopilotpattern/docker-registry.svg)](https://registry.hub.docker.com/u/autopilotpattern/docker-registry/)
+[![DockerStars](https://img.shields.io/docker/stars/autopilotpattern/docker-registry.svg)](https://registry.hub.docker.com/u/autopilotpattern/docker-registry/)
+
+### Hello world example
```

Comment:
I feel like the README is missing a description of the components and how they interact; stating plainly that a Nginx container is proxying to the Registry container, whether connectivity between the Nginx container and Registry is encrypted, etc.